tools/validators/ontology_validator/yamlformat/validator/presubmit_validate_types_lib.py

```diff
@@ -118,6 +130,24 @@ def GetEntityType(self, namespace_name, typename):
       return None
     return self.entity_type_universe.GetEntityType(namespace_name, typename)
 
+  def GetUnitsMapByMeasurement(self, field_name):
+    """Returns a set of possible units by a measurement_field. None if a state.
+
+    Args:
+      field_name: string.
+		"""
+    if not self.unit_universe_reverse_map:
+      print('UnitUniverse undefined in ConfigUniverse')
+      return None
+    subfields = field_name.split('_')
+    # if the last element is numeric need to remove it
+    while subfields[-1].isnumeric():
+      subfields.pop()
+      
+    if subfields[-1] in ['setpoint', 'sensor', 'accumulator', 'command']:
```

Comment:
If you filter point types at all I would do it in the reverse (excluding the ones that you know for sure will never have a unit.  That makes it more future proof.

Reply:
I see, we would need to update in case we added new ones with no units.

Reply:
[`status`, `label`, `mode`, `counter`, 'timestamp`]